Signal/src/ViewControllers/DebugUI/DebugUIMessages.m

```diff
@@ -3390,13 +3390,17 @@ + (SSKEnvelope *)createEnvelopeForThread:(TSThread *)thread
         }
     }();
 
-    SSKEnvelope *envelope = [[SSKEnvelope alloc] initWithTimestamp:timestamp
-                                                            source:source
-                                                      sourceDevice:1
-                                                              type:SSKEnvelopeTypeCiphertext
-                                                           content:nil
-                                                     legacyMessage:nil];
-
+    SSKProtoEnvelopeBuilder *envelopeBuilder = [SSKProtoEnvelopeBuilder new];
```

Comment:
There might be some circumstance where it's useful, but in general I'm opposed to swapping an initializer which (e.g. if we add a new required field) guarantees validity at compile time vs. this builder which won't fail until you run it (if you run it 😬 )